akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/FlowGraphDocTest.java

```diff
@@ -131,7 +134,7 @@ public void demonstrateMatValue() throws Exception {
       (b, fold) -> {
         return new Pair<>(
           fold.inlet(),
-          b.from(b.materializedValue()).via(flatten).out());
+          b.from(b.materializedValue()).via(b.graph(flatten)).out());
```

Comment:
I didn’t want to do this particular rename within this PR because it probably conflicts with a lot of other things—it is on my list to do next, but I’ll wait until our big refactorings are merged.